cypress/support/fragments/finance/ledgers/Ledgers.js

```diff
@@ -58,5 +64,33 @@ export default {
       Button('Delete').click(),
       Button('Delete', { id:'clickable-ledger-remove-confirmation-confirm' }).click()
     ]);
+  },
+
+  searchByStatusUnitsAndName(status, acquisitionUnitsName, ledgerName) {
+    this.selectStatusInSearch(status);
+    cy.do([
+      Accordion({ id: 'acqUnitIds' }).clickHeader(),
+      Selection({ id: 'acqUnitIds-selection' }).open(),
+      SelectionList({ id: 'sl-container-acqUnitIds-selection' }).select(acquisitionUnitsName),
+      SearchField({ id: 'input-record-search' }).fillIn(ledgerName),
+      Button('Search').click(),
+    ]);
+  },
+
+  selectStatusInSearch: (ledgerStatus) => {
+    cy.do(Accordion({ id: 'ledgerStatus' }).clickHeader());
+    switch (ledgerStatus) {
+      case 'frozen':
```

Comment:
I saw code with the same values. I suggest to move value into constants

Reply:
may be you will find the way to put string value into interactor and exclude switch from your code